pwiz_tools/Skyline/TestFunctional/FullScanGraphTest.cs

```diff
@@ -157,6 +157,13 @@ protected override void DoTest()
             TestScale(529, 533, 0, 50);
             ClickChromatogram(33.06, 68.8, PaneKey.PRECURSORS);
             TestScale(452, 456, 0, 300);
+
+            RunUI(() => SkylineWindow.ShowGraphSpectrum(true));

```

Comment:
I don't see how this tests the feature beyond that it doesn't crash. You should be checking for matching coordinates, right?

Reply:
Added scale assertions for the sync in both directions.